storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/KvStoreDatabase.java

```diff
@@ -950,23 +947,42 @@ private UpdateResult doUpdate(final StorageUpdate update) {
       LOG.trace("Committing hot db changes");
       updater.commit();
     }
+
+    if (update.isBlobSidecarsEnabled()) {
+      updateBlobSidecarData(
```

Comment:
Maybe move this a screen above to the previous  `if (update.isBlobSidecarsEnabled()) {` block?

Reply:
yes... in the current form, it is also much safer to write blobs before blocks.